exercises/practice/atbash-cipher/atbash-cipher.ua

```diff
@@ -0,0 +1,5 @@
+# Create an implementation of the atbash cipher
```

Comment:
This comment applies only to the `Encode` function. Maybe add a newline between the encode and decode functions and add function-specific comments?